crates/cfg/src/cfg_expr.rs

```diff
@@ -62,6 +72,69 @@ impl CfgExpr {
         }
     }
 }
+fn next_cfg_expr_from_syntax<I>(iter: &mut Peekable<I>) -> Option<CfgExpr>
```

Comment:
Let's move this stuff into `cfg_process`, it doesn't sit right with me that the `cfg` crate depends on `syntax`.